README.md

```diff
@@ -1 +1,87 @@
-# Technical-Task-Challenges
\ No newline at end of file
+# Technical Task Challenges
+
+Virtual Coffee's Technical Task Challenges are created/aggregated by VC members to help other VC members gain practical experience in how to use technical skills in real-world scenarios.
+
+### Benefits
+
+These task challenges will greatly benefit the entire Virtual Coffee community. They will serve our members who are current working professionals by allowing them to up-skill in select technologies and tools. They will also help our job seekers, especially those with no prior professional experience in the tech industry. The unique added value of the technical task challenges is that they're modeled after real-world scenarios, lending these exercises a quality that far surpasses what's readily available on sites like LeetCode and Codewars.
+
+## Table of Contents
+- [Adding your Experience](#adding-your-experience)
+- [Posting a New Technical Task Challenge](#posting-a-new-technical-task-challenge)
+- [Technical Task Challenge Skill Levels Tracks](#technical-task-challenge-skill-levels-tracks)
+- [Working on a Technical Task Challenge](#working-on-a-technical-task-challenge)
+- [Submitting Your Technical Task Challenge](#submitting-your-technical-task-challenge)
+- [Going Further](#going-further)
+- [Honerable Mentions](#honorable-mentions)
+- [Code of Conduct](./CODE_OF_CONDUCT.md)
+
+### Adding Your Experience
+
+
+As a VC member with real-world experience, you may want to list some of your routine technical tasks. To do this, first create a fork of the repository and a new branch titled `add-to-directory`. Add your name, job title(s), years in the role(s), and tasks to the directory document. Then, make a pull request back to the main project.
```

Comment:
**question:** We say "your routine technical tasks" in here, but Tom Cudd brought up a good point in that not every job in the tech industry is super technical. Do we want to try to stress the technical aspect less to keep it open?

**issue (non-blocking):** We need to add a directory document for people to list their jobs and tasks. Once we do that, we should link it back here. Let's open an issue to do this so we don't forget!